test/pet.test.js

```diff
@@ -0,0 +1,189 @@
+const Pet = require('../src/pet');
+
+let testPet;
+
+const deadMessage = "Your pet is no longer alive :("
+
+beforeEach(() => {
+    testPet = new Pet();
+  });
+
+describe('constructor', () => {
+    it('should return a new pet with a name', () => {
+        expect(new Pet()).toBeInstanceOf(Object);
+        expect(new Pet().name).toBe("An unloved pet");
+        expect(new Pet('Manny').name).toBe("Manny");
+    })
+
+    it('should return a new pet with an age', () => {
+        expect(new Pet().age).toBe(0);
+    })
+
+    it('shold return a new pet with a hunger', () => {
+        expect(new Pet().hunger).toBe(0);
+    })
+
+    it('shold return a new pet with a fitness', () => {
+        expect(new Pet().fitness).toBe(10);
+    })
+})
+
+describe('growUp', () => {
+    it('should make the pet grow', () => {
```

Comment:
This one is a bit too vague. If I came in and had no idea what these tests were for, I have no idea what grow means in this context, so try to stick to something more specific like it should increase the pet's age by 1. Your names want to match the functionality, logic and conditions you are testing for